json-server/config/routes.json

```diff
@@ -4,14 +4,21 @@
     "/tasktypes/deputy*": "/tasktypes-deputy",
     "/users/current": "/users-current",
     "/assignees/:id/clients?*": "/caseload/:id",
+
+
     "/assignees/teams/tasks\\?teamIds[]=21&filter=status\\:Not+started&limit=25&page=:pageNumber&sort=dueDate\\:asc*": "/tasks/21",
     "/assignees/teams/tasks\\?teamIds[]=22&filter=status\\:Not+started&limit=25&page=:pageNumber&sort=dueDate\\:asc*": "/tasks/22",
 
-    "/assignees/teams/tasks\\?teamIds[]=13&filter=status\\:Not+started&limit=25&page=:pageNumber&sort=dueDate\\:asc*": "/tasks-view-25/:pageNumber",
-    "/assignees/teams/tasks\\?teamIds[]=13&filter=status\\:Not+started&limit=50&page=:pageNumber&sort=dueDate\\:asc*": "/tasks-view-50/:pageNumber",
-    "/assignees/teams/tasks\\?teamIds[]=13&filter=status\\:Not+started&limit=100&page=:pageNumber&sort=dueDate\\:asc*": "/tasks-view-100/:pageNumber",
-
     "/assignees/teams/tasks\\?teamIds[]=:teamId&*task_type_category:deputy": "/deputy-tasks/:teamId",
-    "/assignees/teams/tasks\\?teamIds[]=:teamId&*": "/tasks/:teamId",
-    "/users/:id/tasks/:id": "/assign-tasks-to-casemanager"
+    "/users/:id/tasks/:id": "/assign-tasks-to-casemanager",
+
+    "/assignees/teams/tasks\\?teamIds[]=22&filter=status\\:Not+started&limit=25&page=1*": "/tasks-view-25/1",
+    "/assignees/teams/tasks\\?teamIds[]=22&filter=status\\:Not+started&limit=25&page=2*": "/tasks-view-25/2",
+    "/assignees/teams/tasks\\?teamIds[]=22&filter=status\\:Not+started&limit=25&page=3*": "/tasks-view-25/3",
+    "/assignees/teams/tasks\\?teamIds[]=22&filter=status\\:Not+started&limit=25&page=4*": "/tasks-view-25/4",
+    "/assignees/teams/tasks\\?teamIds[]=22&filter=status\\:Not+started&limit=25&page=5*": "/tasks-view-25/5",
+    "/assignees/teams/tasks\\?teamIds[]=22&filter=status\\:Not+started&limit=50&page=1*": "/tasks-view-50/1",
+    "/assignees/teams/tasks\\?teamIds[]=22&filter=status\\:Not+started&limit=100&page=1*": "/tasks-view-100/1",
```

Comment:
I would use a route variable for this

`"/assignees/teams/tasks\\?teamIds[]=22&filter=status\\:Not+started&limit=25&page=:pageNumber*": "/tasks-view-25/:pageNumber"`

Reply:
I tried exactly that but it didnt pick up the route for some reason.

Reply:
This works for me

`"/assignees/teams/tasks\\?teamIds[]=22&filter=status\\:Not+started&limit=:limit&page=:pageNumber&*": "/tasks-view-:limit/:pageNumber",`